spec/helpers/generic_object_definition_helper_spec.rb

```diff
@@ -0,0 +1,32 @@
+class GenericObjectHelperClass
+  include GenericObjectDefinitionHelper
+
+  def initialize(toolbar_builder)
+    @toolbar_builder = toolbar_builder
+  end
+
+  def _toolbar_builder
```

Comment:
This is just for the spec's helper class, it's just so that I can inject the stubbed toolbar_builder when I create a new instance of it.